lib/rabbit/lib/InternalMiddlewareProducer.js

```diff
@@ -0,0 +1,26 @@
+const AbstractFanoutProducer = require('../AbstractFanoutProducer')
+
+class InternalMiddlewareProducer extends AbstractFanoutProducer {
```

Comment:
You do not need extra Producer here.
You can use general profileProducer, please generalize it.
Use routingKey (you use 'user.created' string for your purposes - it is ok)

If we will create a lot of queues we get some kind of "channel hell"